### DISCUSSION TITLE
Visualizer design

### DISCUSSION THREAD

**Discussion by SaadNiaziDev** [samhirtarif/react-audio-recorder#57] [General]
Hey, 
I have been exploring this wonderful repository and used in my project. 
![image](https://github.com/samhirtarif/react-audio-recorder/assets/91617357/26e00072-a174-44f7-8103-8f8f6cc1ad51)
visualizers tend to go upside down from center, but this goes upside donw from the left side.
Like this website is using line visualizer
https://meyda.js.org/
Can we customize it?

**Comment by samhirtarif** [comment]
@SaadNiaziDev - This does seem like a nice-to-have feature. Please feel free to work on this and contribute.
This change would go in the [`react-audio-visualize`](https://github.com/samhirtarif/react-audio-visualize) repository though. The `react-audio-recorder` repo uses `react-audio-visualize` under-the-hood
